### ISSUE TITLE
Integrate shipyard scheduling DVM

### ISSUE THREAD

**Issue by staab** [coracle-social/coracle#305]
Add scheduling to the note options dialog

**Comment by ticruz38** [comment]
In the NoteCreate view, add an option to create a DVM request?

**Comment by staab** [comment]
Right, and display it in NoteOptions

**Comment by ticruz38** [comment]
I need more context for this one issue:
Is the goal just to postpone the posting of a note? In which case I could'nt find a DVM kind or handler for this specific purpose.
Or is the goal creating any kind of DVM request depending on the DVM handlers available from a set of Relays?

**Comment by staab** [comment]
Yep, the goal is to do what shipyard.pub does and allow people to postpone the sending of a note. You can find some information about that DVM kind here: https://www.data-vending-machines.org/kinds/5905/

**Comment by ticruz38** [comment]
> https://www.data-vending-machines.org/kinds/5905/

Everything is clear from this spec expect for the time the note must be dispatched, it's the created_at field that is used?

**Comment by staab** [comment]
On review, this feels potentially scope-creepy to me. I'm surveying people to find out if we should go ahead with this: `nostr:nevent1qvzqqqqqqypzp978pfzrv6n9xhq5tvenl9e74pklmskh4xw6vxxyp3j8qkke3cezqy88wumn8ghj7mn0wvhxcmmv9uq3camnwvaz7tmgdajxccn0vshxxmmjv93kcefww3hk7mrn9uq32amnwvaz7tmjv4kxz7fwv3sk6atn9e5k7tcpzemhxue69uhkzat5dqhxummnw3erztnrdakj7qfhwaehxw309askgmmjd9hxwcmpwfjxjmnpdscjumrwvf5hguewvdhk6tmwdaehgunjv4kxz7f0w3jhxapdwfjkccteqqswkap3kp6lpq4ddcljranwepwnxznmm3actsvqmk5djn3wt9tks2gg34930`

**Comment by ticruz38** [comment]
> Everything is clear from this spec expect for the time the note must be dispatched, it's the created_at field that is used?

I can confirm, the created_at field defines the time the DVM should publish the note
